Sources/Brave/Frontend/Browser/Playlist/Managers & Cache/PlaylistCarplayManager.swift

```diff
@@ -72,23 +72,13 @@ public class PlaylistCarplayManager: NSObject {
     // Setup Playlist Download Resume Session
     PlaylistManager.shared.restoreSession()
 
-    // REFACTOR to support multiple windows.
-    // OR find a way to get WebKit to load `Youtube` and other sites WITHOUT having to be in the view hierarchy..
-    var currentWindow = browserController?.view.window
-
-    // BrowserController can actually be null when CarPlay is launched by the system
-    // The only such window that will be available is the actual CarPlay window
-    // So that's the window we need to use
-    // This does NOT break Multi-Window because we don't actually have multiple windows running
-    // when there is no browser controller. There is only a single CarPlay window.
-    if currentWindow == nil {
-      currentWindow =
-        UIApplication.shared.connectedScenes
-        .filter({ $0.activationState == .foregroundActive })
-        .compactMap({ $0 as? UIWindowScene })
-        .first?.windows
-        .filter({ $0.isKeyWindow }).first
-    }
+    // REFACTOR to find a way to get WebKit to load `Youtube` and other sites WITHOUT having to be in the view hierarchy..
```

Comment:
Ahh yeah this was written in Playlist v1 when CarPlay was added for the first time. I'm really not sure what to do here yet because CarPlay can only interact with a SINGLE browser instance technically. The thing is, iPad doesn't even support CarPlay, so this refactor is only true for iPhone which only supports a single window (though this can change).

But on iOS 17+, it can use the new APIs to allow WebViews to load in the background WITHOUT a browser "window" (IE: It doesn't have to be in the view hierarchy).

I guess I could just make a "window" for CarPlay only, and have it never interact with any "browser", but I think I will tackle this later or see what I can do on iOS 17+ first.

I will make a ticket.